src/server/routers/user.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const { 
+    postRegisterDetails 
+} = require("../controllers/user");
+
+const router = express.Router();
+
+router.post("/register", postRegisterDetails);
```

Comment:
Once we have more controllers, I think we should review our route/function names

Reply:
Yes, I think as our App develops we can review our function names and improve on them.